### PR TITLE
Rasa X Starter Pack Material

### PR DESCRIPTION
**Pull request by b-quachtran**
Added an SQLite DB pre-loaded with two conversations and the tag `nlu-issue` for use with the Rasa X starter pack material

### CODE REVIEW

**Comment by akelad** [comment]
@melindaloubser1 or @erohmensing would either of you be up for taking on this PR?

**Comment by melindaloubser1** [comment]
sure can do

File: README.md [hunk 1]
```diff
@@ -227,6 +227,8 @@ To [deploy financial-demo](https://rasa.com/docs/rasa/user-guide/how-to-deploy/)
 [one line deploy script](https://rasa.com/docs/rasa-x/installation-and-setup/one-line-deploy-script/) for Rasa X. As part of the deployment, you'll need to set up [git integration](https://rasa.com/docs/rasa-x/installation-and-setup/integrated-version-control/#connect-your-rasa-x-server-to-a-git-repository) to pull in your data and
 configurations, and build or pull an action server image.
 
+For running Rasa X in local mode, you can refer to our guided workflow in the [Wiki page](https://github.com/RasaHQ/financial-demo/wiki/Using-Rasa-X-with-the-Financial-Demo) for how to get started.
+
```
Comment (melindaloubser1) [inline README.md:231]: How about making a note of this closer to the top? Something like "You can try out the bot locally, or try it out in Rasa X with pre-loaded conversations!" Under "run the bot"

Comment (melindaloubser1) [reply README.md:231]: A link to the install instructions for local mode would also be helpful - or you could even include the one-liner in the readme

**Review by melindaloubser1 — COMMENTED** [review]
In the wiki (can't comment directly),  it'd be good to have a bit of an intro section "Rasa X is a tool for Conversation Driven Development. (link to blog post) You can try it out with the Financial Services Bot to see how it works, with some conversations already loaded. Besides talking to the bot, you can annotate messages, save conversations as stories and tests, and share the bot with others (link to share with ngrok)"

**Comment by melindaloubser1** [comment]
probably because of time zones, I see Aug 11, not 10, and a different time. Might need to refer to convos by other means - maybe pre-tag them?

**Comment by melindaloubser1** [comment]
I think adding some more convos would be good - maybe a few more with NLU issues, so you can refer to them e.g. "You'll notice some NLU issues in other conversations. Try annotating the messages that weren't predicted correctly to improve your model"

**Comment by b-quachtran** [comment]
I added a flag to the first NLU issue and re-worded the wiki to refer to the next conversation as opposed to timestamp. I don't really like the idea of referring to the conversation by tag since tagging is part of the workflow

**Comment by melindaloubser1** [comment]
Good point, could be confusing to use tags

File: README.md [hunk 2]
```diff
@@ -58,11 +58,18 @@ Then to talk to the bot, run:
...
 Note that `--debug` mode will produce a lot of output meant to help you understand how the bot is working
 under the hood. To simply talk to the bot, you can remove this flag.
 
 
+You can also try out your bot locally using Rasa X by running
+```
+rasa x
+```
+
+Refer to our guided workflow in the [Wiki page](https://github.com/RasaHQ/financial-demo/wiki/Using-Rasa-X-with-the-Financial-Demo)for how to get started with Rasa X in local mode.
```
Comment (melindaloubser1) [inline README.md:70]:
```suggestion
Refer to our guided workflow in the [Wiki page](https://github.com/RasaHQ/financial-demo/wiki/Using-Rasa-X-with-the-Financial-Demo) for how to get started with Rasa X in local mode.
```

**Review by melindaloubser1 — APPROVED** [review]

**Comment by melindaloubser1** [comment]
Looks good!

**Pull request merged**